app/views/posts/_form.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= form_for [topic, post] do |f| %>
+  <% if post.errors.any? %>
+    <div class="alert alert-danger">
+      <h4>There are <%= pluralize(post.errors.count, "error") %>.</h4>
```

Comment:
How will this line read if there is one error? How could you make it more readable?